.buildkite/deploy.yml

```diff
@@ -0,0 +1,13 @@
+steps:
+  - label: ":package: Build image"
+    key: build
+    command:
+      - "docker build -t ${CONTAINER_NAME} -f ./${CONTAINER_NAME}/Dockerfile ."
+  - label: ":docker: Publish to docker hub"
+    depends_on: build
+    command:
+      - "docker tag ${CONTAINER_NAME} tapendium/${BUILDKITE_TAG}"
+      - "docker push tapendium/${BUILDKITE_TAG}"
+    plugins:
+      - docker-login#v2.0.1:
+          username: "${DOCKER_LOGIN_USERNAME}"
```

Comment:
**question:** Does reading from the environment variable work here?

Reply:
not sure let me try it locally and see